documentation/Components/LayoutSample.fsx

```diff
@@ -1,11 +1,90 @@
 #load "../../.paket/load/netstandard2.0/main.group.fsx"
 #load "../../src/Common.fs"
-#load "../../src/Layout.fs"
+#load "../../src/Container.fs"
+#load "../../src/Row.fs"
+#load "../../src/Col.fs"
 
 open Fable.Core.JsInterop
 open Fable.React
+open Fable.React.Props
 open Reactstrap
 
-let private layoutSample = FunctionComponent.Of<obj>((fun _ -> fragment [] []), "LayoutSample")
+let container = Container.container
```

Comment:
I like the idea of aliases but wish to keep things as simple as possible in the examples.
I rather have the long version `Row.row` just so that works if it is copy-pasted.